app/controllers/admin/member_search_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::MemberSearchController < Admin::ApplicationController
+  def index
+    @params = params[:member_search] || {}
+    @name = @params[:name]
+    @members = @name.blank? ? Member.none : Member.find_members(@name).select(:id, :name, :surname, :pronouns)
+    @callback_url = @params[:callback] || results_admin_member_search_index_path
+    if @members.size == 1
+      query = { member_pick: { members: [@members.first.id] } }
+      query_string = query.to_query
+      @callback_url = "#{@callback_url}?#{query_string}"
+      redirect_to @callback_url and return
+    end
+
+    render 'index', locals: { members: @members, callback: @callback_url }
```

Comment:
```suggestion
    render 'index', locals: { members: @members, callback_url: @callback_url }
```

Reply:
So, the @callback_url can also become a local variable, not an instance variable.